data/json/items/ammo/223.json

```
@@ -127,6 +127,15 @@
     "ascii_picture": "556_incendiary",
     "extend": { "effects": [ "INCENDIARY" ] }
   },
+  {
+    "id": "556_ap",
+    "copy-from": "556",
+    "type": "AMMO",
+    "name": { "str_sp": "5.56 NATO M995" },
+    "description": "An M995 armor-piercing round using a 52gr projectile, featuring a black-tipped, tungsten carbide penetrator over an aluminum slug.  Higher material penetration in exchange of lower overall damage.",
```

```suggestion
    "description": "An armor piercing 5.56x45mm M995 round using a 52gr projectile, featuring a black-tipped, tungsten carbide penetrator over an aluminum slug.  Increased penetration over standard ball rounds at the cost of worse wounding potential in soft tissue.",
```
Damage is a game mechanic and shouldn't be used that way in an item description.

---

[<>](https://en.wikipedia.org/wiki/Damage)

---

It is a valid word but in the context it looks like it refers to the item stat and we don't like doing that in descriptions.